### DISCUSSION TITLE
Image Component not working in AMP mode

### DISCUSSION THREAD

**Discussion by Glomels** [vercel/next.js#18442] [Help]
I could not find any information regarding AMP support for the new Image component, so I'm not sure if this is supposed to work or not.

I tried using it in an AMP page and the image doesn't render, so I'm guessing it's not supported yet?

**Comment by M1ck0** [comment]
By looking at [this](https://nextjs.org/docs/api-reference/next/amp) AMP page has `<amp-img>` tag for images so I think that you have to use that for showing images on AMP pages.

**Comment by Glomels** [comment]
I wasn't really clear but I'm talking about the new Image component that was just released in version 10 yesterday.
I'd like to leverage the automatic image optimization. Most of my sites are AMP based.
I haven't found any information in how to use the new Image component in AMP mode.
